app/controllers/AuthorizedToViewController.scala

```diff
@@ -69,56 +69,180 @@ class AuthorizedToViewController @Inject()(authenticate: IdentifierAction,
       }
   }
 
-  private def getCsvFile(eori: String)(implicit req: AuthenticatedRequest[_]): Future[Seq[StandingAuthorityFile]] = {
-    sdesConnector.getAuthoritiesCsvFiles(eori)
-      .map(_.sortWith(_.startDate isAfter _.startDate).sortBy(_.filename).toSeq.sortWith(_.filename > _.filename))
-  }
-
   def onSubmit(): Action[AnyContent] = authenticate async { implicit request =>
     form.bindFromRequest().fold(
       formWithErrors =>
         for {
           csvFiles <- getCsvFile(request.user.eori)
         } yield {
-          val viewmodel = csvFiles
+          val viewModel = csvFiles
           val fileExists = csvFiles.nonEmpty
-          val url = Some(viewmodel.headOption.map(_.downloadURL).getOrElse(""))
-          val date = Formatters.dateAsDayMonthAndYear(Some(viewmodel.headOption.map(_.startDate).getOrElse(LocalDate.now)).get)
+          val url = Some(viewModel.headOption.map(_.downloadURL).getOrElse(""))
+          val date = Formatters.dateAsDayMonthAndYear(Some(viewModel.headOption.map(_.startDate).getOrElse(LocalDate.now)).get)
           BadRequest(authorisedToViewSearch(formWithErrors, url, date, fileExists))
         },
-      query => {
-        val searchQuery = stripWithWhitespace(query)
-
-        val maxWaitTime: FiniteDuration = Duration(5, TimeUnit.SECONDS)
-        val isMyAcc = Await.result(apiService.getAccounts(request.user.eori).map(
-          _.myAccounts.exists(_.number == query)), maxWaitTime)
-
-        if (request.user.eori.equalsIgnoreCase(query)) {
-          Future.successful(BadRequest(authorisedToViewSearch(form.withError("value",
-            "cf.account.authorized-to-view.search-own-eori").fill(query),
-              Some(""), LocalDate.now.toString, false)))
-         }  else if (isMyAcc) {
-            Future.successful(BadRequest(authorisedToViewSearch(form.withError("value",
-              "cf.account.authorized-to-view.search-own-accountnumber").fill(query),
-                Some(""), LocalDate.now.toString, false)))
+      query => processSearchQuery(request, query)
+    )
+  }
+
+  private def processSearchQuery(request: AuthenticatedRequest[AnyContent],
+                                 query: EORI)(
+                                  implicit hc: HeaderCarrier,
```

Comment:
this spacing looks slightly indented here?

Reply:
I crossed check and its displaying fine for me. Hence resolving